src/skmatter/linear_model/_ridge.py

```diff
@@ -110,6 +112,9 @@ def __init__(
         self.shuffle = shuffle
         self.n_jobs = n_jobs
 
+    def _more_tags(self):
+        return {"multioutput_only": True}
```

Comment:
Shouldn't it be `multioutput`, since we also support one output? I don't really understand what sklearn means with this and I could not find doc. Is this just about the shape of the output?

Reply:
This is just about the shape of the output